app/config/galoy-instances.ts

```diff
@@ -0,0 +1,30 @@
+import { INetwork } from "@app/types/network"
+import { scriptHostname } from "@app/utils/helper"
+
+export type GaloyInstance = {
+  name: "BBW" | "Staging" | "Local"
+  network: INetwork
+  graphqlUri: string
+  graphqlWsUri: string
+}
+
+export const GALOY_INSTANCES: GaloyInstance[] = [
```

Comment:
maybe a follow up PR, but would be a good idea to be able to add new instance, even if just from the developer screen initially. there are things related to the federation that we could test out that way.

Reply:
I will create an issue for a follow up PR.